yarn.lock

```diff
@@ -514,7 +514,7 @@
     "@types/warning" "^3.0.0"
     warning "^4.0.3"
 
-"@iota/types@^1.0.0-beta.9":
```

Comment:
Not sure this needs to be in here?

Reply:
It shouldn't be, but I think it's here because of [this line](https://github.com/iotaledger/firefly/blob/stardust-develop/package.json#L9) in `stardust-develop`. I can remove or update that dev dependency to help clear it up?